api/apps/articles/model/save.coffee

```diff
@@ -217,15 +217,23 @@ denormalizedArtworkData = (artwork) ->
     artist:
       name: getArtistName artwork
       slug: artwork.get('artist')?.id
+    artists: getArtistsNames artwork
 
 getArtistName = (artwork) ->
   if artwork.get('artist')?.name
     artwork.get('artist').name
-  else if artwork.get('artists')?.length > 0
-    _.compact(artwork.get('artists').pluck('name'))[0]
   else
     ''
 
+getArtistsNames = (artwork) ->
+  artists = []
+  if artwork.get('artists')?.length > 0
+    artwork.get('artists').forEach (artist) ->
+       artists.push({name: artist.name, slug: artist.id})
+  else
```

Comment:
If `artist` is deprecated on Gravity, I wonder if we should even bother using it. Maybe this would just assume that if there is nothing in `artists`, we could just use the empty state `artists: []`.